Harbor/PreferencesPaneWindowController.swift

```diff
@@ -142,4 +142,24 @@ class PreferencesPaneWindowController: NSWindowController, NSWindowDelegate, NST
   private func enableOrDisableSaveButton() {
     savePreferencesButton.enabled = codeshipAPIKeyError.stringValue.isEmpty && refreshRateError.stringValue.isEmpty
   }
+
+  private func convertIntToBoolean(loginState: Int) -> Bool {
```

Comment:
cool, we'll work on that this afternoon


On Thu, Jul 14, 2016 at 1:23 PM, Ty Cobb <notifications@github.com> wrote:

> In Harbor/PreferencesPaneWindowController.swift
> <https://github.com/devmynd/harbor/pull/10#discussion_r70857270>:
>
> > @@ -142,4 +142,24 @@ class PreferencesPaneWindowController: NSWindowController, NSWindowDelegate, NST
> >    private func enableOrDisableSaveButton() {
> >      savePreferencesButton.enabled = codeshipAPIKeyError.stringValue.isEmpty && refreshRateError.stringValue.isEmpty
> >    }
> > +
> > +  private func convertIntToBoolean(loginState: Int) -> Bool {
>
> I take back what I said about the extension.
>
> I think it makes sense to add an extension var to NSButton like var on:
> Bool(or checked or something) that updates the button's state to either
> NSOnState or NSOffState (I'd also use those named values instead of 1/0).
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/devmynd/harbor/pull/10/files/7fb49049dbdb0bc95156705f0d49bd0f2e55633e..d23ee3a2b0bdacaf0fd53deb69ff9d4f41345136#r70857270>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAB4jgRSwd65H5bLYgL3Pj3HY_UBkr6Xks5qVn6-gaJpZM4JMkgm>
> .
>